apps/starter/src/ensemble/screens/dashboard.yaml

```diff
@@ -0,0 +1,425 @@
+View:
```

Comment:
Do not add more screens to starter project. Add your example usage to the kitchen sink